apps/src/lib/node/ledger/protocol/transactions/validator_set_update/mod.rs

```diff
@@ -0,0 +1,20 @@
+//! Code for handling
+//! [`namada::types::transaction::protocol::ProtocolTxType::ValidatorSetUpdate`]
+//! transactions.
+
+use eyre::Result;
+use namada::ledger::storage::traits::StorageHasher;
+use namada::ledger::storage::{DBIter, Storage, DB};
+use namada::types::transaction::TxResult;
+use namada::types::vote_extensions::validator_set_update;
+
+pub(crate) fn aggregate_votes<D, H>(
+    _storage: &mut Storage<D, H>,
+    _ext: validator_set_update::VextDigest,
+) -> Result<TxResult>
+where
+    D: 'static + DB + for<'iter> DBIter<'iter> + Sync,
+    H: 'static + StorageHasher + Sync,
+{
+    todo!()
```

Comment:
This could cause the ledger to panic since this code path can be called right now, we could log a warning saying nothing will happen yet and return an `Ok(TxResult::default())`. Also I'm not sure why CI didn't run for the most recent commit on this PR, but I think that would catch this as well